README.md

```diff
@@ -170,9 +167,12 @@ Any supply name placed in this array will be excluded from the allowed supplies
 #### Eval Mode
 When not running in training mode (-t), additional checks are implemented:
 * No supplies or treatments are allowed that are not in the simulator. Put the names of these treatments in the trainingOnlySupplies array in dependencies.json
+* No blanket can appear on the character at the start. 
 
 #### Injury/Location Matches
-Injuries are only allowed to have specific locations. Please follow the table to create valid matches.
+* Only a maximum of 8 injuries can be shown on the simulation, not counting ear bleed, asthmatic, broken bones, and
+internal injuries.
+* Injuries are only allowed to have specific locations. Please follow the table to create valid matches. 
```

Comment:
What will it be instead?

Reply:
Nothing I think?  They can do forearm and wrist.  Or I guess we can add left/right arm to the API.
In theory this is supposed to match what's possible in the sim, so if you see anything that looks off, we can address it.  I think this list started off as a "best guess."  This list also was written prior to the training flag.

Reply:
@Anchita14 , in Teams discussion with @kaitlyn-sharo , it turns out that "arm" was supposed to be "hand" (my bug).  But we don't have burns on hands, so that should be removed.  I'll go ahead and push that small fix, then I think this ticket is good to go!

As long as these little fixes are minor, it's generally okay to do them in unrelated PRs.  This one's just a documentation fix.  But when these things start touching multiple files or involve complicated logic, a new ticket should be written.